### PR TITLE
reading urls from token

### PR DESCRIPTION
**Pull request by karthikbr82**

### CODE REVIEW

File: callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  /** The connection time out. */
+  public static int connectionTimeOut = CallStatsConst.CONNECTION_TIMEOUT;
+
+  public static void init() {
+    Properties prop = new Properties();
+    InputStream input = null;
+
+    try {
+      input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
```
Comment (eljasala) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java]: We are reading this in quite a few places, would it be better to move at least reading to single place?

File: callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+public class CallStatsConfigProvider {
+
+  /** The Constant logger. */
+  private static final Logger logger = LogManager.getLogger("CallStatsConfigProvider");
+
+
+  /** The base url. */
+  public static String EVENTS_BASE_URL;
+  public static String STATS_BASE_URL;
+  public static String AUTH_BASE_URL;
```
Comment (eljasala) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java]: These probably shouldn't be capitalized as they are not constants.

File: callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java (hunk 1 above)
Comment (karthikbr82) [reply callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java]: Yes I removed in other places and created this new class.

Comment (eljasala) [reply callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java]: It's still done here: https://github.com/callstats-io/callstats.java/blob/b7b5134798c3a96ec40bc27817cf685015314960/callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java#L80

**Comment by eljasala** [comment]
Also, can you change the version that is sent to collector? I think it's currently hardcoded to 4.0.0.

**Pull request merged**